apps/builder/app/builder/features/style-panel/style-source/style-source-input.tsx

```diff
@@ -428,7 +449,20 @@ export const StyleSourceInput = (
     },
   });
 
-  const inputProps = getInputProps();
+  const inputProps = getInputProps({
+    onKeyDown(event) {
+      if (
+        event.key === "Backspace" &&
```

Comment:
it was removed because the local token was last and was not removable, so we had to remove this, but now I copied it back from the old implementation, because now its good again